Add an explainer for the UndoManager API.

whsieh opened the pull request:
* UndoManager/README.md: Added.

Add the explainer.

* UndoManager/demo/drawing.js: Added.
* UndoManager/demo/editing.js: Added.
* UndoManager/demo/index.css: Added.
* UndoManager/demo/index.html: Added.
* UndoManager/demo/index.js: Added.

Add a simple demo page that uses the experimental UndoManager API.
This demo page implements a simple drawing web application, where
each stylus stroke is undo-able and redo-able in a way that
integrates with system undo/redo affordances as well (e.g. iPad
undo/redo arrow buttons).

* UndoManager/edit-menu-bar-macOS.png: Added.
* UndoManager/shake-to-undo-iOS.png: Added.
* UndoManager/undo-system-ui-iOS.png: Added.

Add example screenshots (referenced in the explainer's markdown).

hober reviewed (changes requested):
Please also add a link to the new explainer to the README.md in the root directory. Thanks!

whsieh commented:
> Please also add a link to the new explainer to the README.md in the root directory. Thanks!

Thanks for taking a look! I added a link to `UndoManager/` in the updated PR.
I also added a quick recording of the demo page as well in this updated version.

hober reviewed (approved):
Great work!
